src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2858,10 +2858,11 @@ static WellKnownMembers()
                 (byte)(MemberFlags.Method | MemberFlags.Static),                                                                                    // Flags
                 (byte)WellKnownType.ExtSentinel, (byte)(WellKnownType.Microsoft_CodeAnalysis_Runtime_Instrumentation - WellKnownType.ExtSentinel),  // DeclaringTypeId
                 0,                                                                                                                                  // Arity
-                    4,                                                                                                                              // Method Signature
+                    5,                                                                                                                              // Method Signature
```

Comment:
This change is fine given that v1 hasn't shipped, but raises a question. 
Has anyone given thought about how the interface between compiler and instrumentation runtime will be versioned or maintained in the future?
I'm assuming that the compiler and instrumentation runtime won't always necessarily be the same version, is that right?